README.md

```diff
@@ -53,6 +53,9 @@ sh build/build.sh
 # when a change is detected.
 sh build/autobuild.sh
 
+# Configure the webpage directory
```

Comment:
This appears to be an optional step for a local build.  If `sh build/build.sh` is still the minimal command needed to build the PDF and HTML manuscripts, we may want to indicate that more explicitly in the script comments or preceding text.